components/centraldashboard/public/components/main-page.pug

```diff
@@ -1,5 +1,7 @@
 iron-ajax(auto, url='/api/platform-info', handle-as='json',
     on-response='_onPlatformInfoResponse')
+iron-ajax(auto, url='/api/user-info', handle-as='json',
```

Comment:
I think when we decide to implement real support for obtaining user identity, we should consider providing a new server endpoint that returns the platform information, list of namespaces, and user identity in a single message. With this change, we'd be issuing 3 AJAX calls after the page loads (Chrome has a limit of 6 concurrent per hostname) to initialize the state of the page so it would probably be a good idea to consolidate. See my previous comment in the server code about tabling this until we have a working identity attainment mechanism.

Reply:
Makes sense, stashed this change